### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by skrolikowski** [fusioncms/fusioncms#320] [Ideas]
There's only one language for FusionCMS: English. It would be nice to add the ability for users to contribute multiple languages for their content.

Language support is provided by Laravel, however this is for simple messaging:
https://laravel.com/docs/6.x/localization

**Comment by mustafa-online** [comment]
**+1** for this.

**Comment by manda-ikaink** [comment]
In a pinch, I was able to follow this tutorial to add basic localization to Laravel to support multi-language on a **theme** specific level. Language files were added to the theme's `lang` directory and could be switched successfully. Doesn't account for content though. 

https://dev.to/fadilxcoder/adding-multi-language-functionality-in-a-website-developed-in-laravel-4ech

**Comment by skrolikowski** [comment]
A neat way to spin-up lang file records:
https://laravel-news.com/extract-untranslated-strings-from-laravel-projects
